apps/bridgemail/campaigns/campaign_step1.js

```diff
@@ -370,18 +370,24 @@ function (template) {
                     var merge_field_patt = new RegExp("{{[A-Z0-9_-]+(?:(\\.|\\s)*[A-Z0-9_-])*}}","ig");
                     
                     if(this.$('#campaign_subject').val() == '')
-                    {            
+                    {         
+                        if(!this.parent.$el.find('#accordion_setting').accordion( "option", "active" )){
```

Comment:
@umairmakesbridge  validating if first accordian is closed if so just showing that back again